onnxruntime/test/providers/cpu/generator/random_test.cc

```diff
@@ -104,7 +104,8 @@ TEST(Random, RandomUniform1DFloat) {
 
   test.AddOutput<float>("Y", dims, expected_output);
 
-  test.Run();
+  // TensorRT does not support manual seed overrides and there will be result mismatch
```

Comment:
could this case be detected in GetCapability or TRT Parser?

Reply:
Discussed with Nvidia. Random seed is the normal case that they will support. They don't support manual seed, which will be treated as a warning, not error.